packages/react-core/src/components/Select/Select.tsx

```diff
@@ -1319,6 +1323,7 @@ export class Select extends React.Component<SelectProps & OUIAProps, SelectState
                   aria-activedescendant={typeaheadActiveChild && typeaheadActiveChild.id}
                   id={`${selectToggleId}-select-typeahead`}
                   aria-label={typeAheadAriaLabel}
+                  aria-describedby={typeAheadAriaDescribedby}
```

Comment:
It might be worth following the `aria-describedby` prop here so that the attribute doesn't get added if the prop isn't passed in:

```suggestion
                  {...(typeAheadAriaDescribedby && { 'aria-describedby': typeAheadAriaDescribedby })}
```